### PR TITLE
[api][SIMS-#227] Update Creating of Institution User API

### PR DESCRIPTION
**Pull request by fwpushan**
(#227 )
------
1. Add user info to payload
2. Adding actual user while creating IU
3. Update tests
--

### CODE REVIEW

File: sources/packages/api/src/route-controllers/institution/institution.controller.ts [hunk 1]
```diff
@@ -133,20 +133,26 @@ export class InstitutionController extends BaseController {
       );
     }
 
+    // Create User
+    const userEntity = this.userService.create();
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/route-controllers/institution/institution.controller.ts:137]: I believe that we should ensure that this user belongs to the institution that he is being added to.
I would recommend executing a bceidService.getAccountDetails to ensure the relationship on BCeID and we can also get the firstName, lastName from BCeID response.

**Review by andrewsignori-aot — COMMENTED** [review]
Please take a look at the comment about the user creation.

File: sources/packages/api/src/route-controllers/institution/institution.controller.ts (hunk 1 above)
Comment (fwpushan) [reply sources/packages/api/src/route-controllers/institution/institution.controller.ts:137]: are we getting bceid user name?

Comment (fwpushan) [reply sources/packages/api/src/route-controllers/institution/institution.controller.ts:149]: Resolved

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Review by ann-aot — APPROVED** [review]

**Pull request merged**